bids/modeling/statsmodels.py

```diff
@@ -541,6 +541,7 @@ def merge_dfs(a, b):
         #   the intercept column is given the same name as the input contrast.
         #   It may already exist, in which case we do nothing.
         # * Otherwise, we name the column 'intercept'.
+        int_name = None
         if 1 in var_names:
```

Comment:
Could 1 be a string `"1"` in var_names?

Reply:
By the spec, it is specifically the number 1. I would be okay with doing a separate PR to detect "1" and warn users, but I'd rather not complicate this one.